### DISCUSSION TITLE
Increment/decrement numeric field when scrolling

### DISCUSSION THREAD

**Discussion by Kartopod** [godotengine/godot-proposals#11457] [Editor]
Affinity designer allows you to use your mouse wheel while hovering over a numeric input field to increment/decrement the value by 1 unit. I find that this is very handy because it is precise while not needing the user to touch their keyboard, or selecting the input field at all. This would be very useful to increment fields that are usually small (Such as the font size of a label, or the border size of a stylebox for example). 

![Scroll](https://github.com/user-attachments/assets/ed833ea8-385b-408d-ab00-a687f39fdf0d)

To make sure that the user doesn't accidentally change the value when scrolling elsewhere in the editor, this should only work when the mouse is directly on top of the input field value. Scrolling when a property is clicked on (highlighted) should have no effect.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/2099.
